lib/rules/no-object-constructor.js

```diff
@@ -93,16 +160,32 @@ module.exports = {
             const variable = getVariableByName(sourceCode.getScope(node), "Object");
 
             if (variable && variable.identifiers.length === 0) {
-                const replacement = needsParentheses(node) ? "({})" : "{}";
+                let replacement;
+                let fixText;
+                let messageId;
```

Comment:
It seems like you could initialize this to `"useLiteral"` and then just skip the assignments on lines 174 and 178?

Reply:
Done.